molecule/eval/converge.yml

```
@@ -0,0 +1,113 @@
+---
+
+## EVAL FROM FILE
+
+- name: Eval function from file
+  hosts: cluster
+  roles:
+    - ansible-cartridge
+  become: true
+  become_user: root
+  gather_facts: false
+  vars:
+    cartridge_app_name: 'myapp'
+    cartridge_scenario:
+      - eval
+    cartridge_eval_file: some_script.lua
+    cartridge_eval_args:
+      - Hello, my name is %s!
```

And the same comment below:
```suggestion
      - 'Hello, my name is %s!'
```